pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -22,7 +22,8 @@
     "test": "jest --watch",
     "start": "npm-run-all -p build:watch tsc:watch",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "bump": "npm version prerelease --preid prerelease --no-git-tag-version"
```

Comment:
Same as from https://github.com/cockroachdb/cockroach/pull/91305#pullrequestreview-1169134427 — curious why we're using `npm` here instead of `yarn`, since I imagine someone will eventually get confused 😃